local/src/core/http.cc

```diff
@@ -1587,6 +1705,16 @@ Session::do_connect(TextView interface, swoc::IPEndpoint const *real_target)
           static const int ONE = 1;
           setsockopt(socket_fd, IPPROTO_TCP, TCP_NODELAY, &ONE, sizeof(ONE));
           if (0 == ::fcntl(socket_fd, F_SETFL, fcntl(socket_fd, F_GETFL, 0) | O_NONBLOCK)) {
+            if (pp_version != ProxyProtocolVersion::NONE) {
+              // The following sleep is to allow the client to postpone
+              // initiating the SSL handshake later. It's observed previously
+              // that sometimes the client would timeout and close the
+              // connection.  Commenting it out works fine currently so this
+              // might be resolved but just leaving it in here in case this
+              // issue arise in the future.
+              // std::this_thread::sleep_for(std::chrono::seconds(1));
+              send_proxy_header(real_target, pp_version);
+            }
```

Comment:
This is concerning. :)

Be aware that we will never be able to add a 1 second sleep for establishing connections. It would tank our prod sim throughput and add an annoying delay even for the less time sensitive AuTests.

Reply:
My bet is that this is no longer a issue as it happened way before I finalized my implementation. I will remove the ominous comment for 3 reasons:
* it most likely is no longer relevant in the latest implementation
* the suggested `sleep_for` is not a viable solution even if the issue truly persists, as you suggested
* if the issue arises again, it would be obvious enough to pinpoint this area even without this comment.